android/java/org/chromium/chrome/browser/sync/settings/BraveManageSyncSettings.java

```diff
@@ -49,6 +48,9 @@ public class BraveManageSyncSettings extends ManageSyncSettings {
     private static final String VERBOSE_SYNC_PASSWORDS_PREF_COMMAND_LINE_KEY =
             "verbose_sync_passwords_pref";
 
+    public static final String ARC_FEATURE = "org.chromium.arc";
```

Comment:
I had to [look that up](https://en.wikipedia.org/wiki/Google_App_Runtime_for_Chrome) to figure out what ARC stands for.
```suggestion
    // Android Runtime for Chrome
    public static final String ARC_FEATURE = "org.chromium.arc";
```

Reply:
thanks, applied the suggestion